golden_response/staging/get_pop_obs/moscow.json

```diff
@@ -1837,7 +1837,7 @@
           "observation_date": "2005",
           "observation_period": "P1Y",
           "stat_var_ids": [
-            "LifeExpectancy_Person_Female"
+            "dc/3bpjn5e953v0b"
```

Comment:
I've not been able to spot a visual difference between
https://datacommons.org/browser/LifeExpectancy_Person_Female
https://datacommons.org/browser/dc/3bpjn5e953v0b , and same for Male below; which are all currently in prod. Not sure why the stats got attached to the hash DCID instead of the human readable one-- does this have anything to do with import, or is it more an API behavior?

Reply:
This api directly reads from the popobs cache, so this should happened in flume. Is there any randomness in the statvar picking? But we never seen this before though.

Reply:
OK, if its a flume thing, could be something I introduced. So, let me try the Tues cache...

Reply:
Wait, we shouldn't be looking at the browser, rather we should look at the new cache/BQ.  The curated version has two new properties -- description, descriptionUrl.  I bet its because descriptionUrl is not listed in PropsThatAreNotConstraintProps().

To confirm, you can run the below cmd before and after cl/345715815, and you can see the DCID change from dc/3bpjn5e953v0b to dc/f3w8mmhekf9sf.

  # blaze test -c opt :statvar_checker_test --test_arg=--curated_statvar_id=LifeExpectancy_Person_Female

So, if LifeExpectancy_Person_Female and LifeExpectancy_Person_Male aren't used by any client, we can roll this out. Otherwise we should rework cl/345715815.

Sorry for missing this during review.

Reply:
I'll make a change to add descriptionUrl to PropsThatAreNotConstraintProps().

Reply:
Ah ok, if we make the change to PropsThatAreNotConstraintProps, there is no rework needed for cl/345715815? I'll check if it's used on surfaces now.

Reply:
@shifucun -- does the mixer have such a list? Or does it rely on constraintProps?

Reply:
It depends on constraint props

Reply:
Great, then just a pipeline change should suffice.